tests/test_skvbc_auto_view_change.py

```diff
@@ -77,13 +78,14 @@ class SkvbcAutoViewChangeTest(unittest.TestCase):
         initial_primary = 0
         bft_network.stop_replica(initial_primary)
 
+        skvbc = kvbc.SimpleKVBCProtocol(bft_network)
+
         # do nothing - just wait for an automatic view change
-        await bft_network.wait_for_view_change(
-            replica_id=random.choice(
-                bft_network.all_replicas(without={initial_primary})),
-            expected=lambda v: v > initial_primary,
-            err_msg="Make sure automatic view change has occurred."
-        )
+        await bft_network.wait_for_view(replica_id=random.choice(
+            bft_network.all_replicas(without={initial_primary})), expected=lambda v: v > initial_primary,
+            err_msg="Make sure automatic view change has occurred.")
```

Comment:
again, why are you changing the formatting here? I'd only expect to see the function name change